### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by farishkash** [TKennedy1/TeaCozy#5]
Overall met expectations.

HTML and CSS are organized and well structured.

Alt text should be use for all images(not background as it isn't possible) and also for images when you build a mobile view for your site.

Comments for organization is a best practice especially when the projects start to scale.

Beware of defaults settings in browsers for example with your issue, most browsers give a body a default margin between 6 to 8px;

I really push efficiency in the css.  I have seen too many projects were they have used double to tripe of the amount of css needed to get the job done.  The less you type the less likely you will make an error.  It is also easier to find the error since you have less to read through.

To prepare you for the final projects, look at media queries. See if you can make this site fit inside of a mobile device by using one. You will be doing this heavily in your final projects.

https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries

https://www.w3schools.com/css/css3_mediaqueries_ex.asp
